controllers/healthcheck_controller.go

```diff
@@ -632,6 +636,22 @@ func (r *HealthCheckReconciler) parseWorkflowFromHealthcheck(log logr.Logger, hc
 		log.Error(err, "Invalid spec file passed")
 		return err
 	}
+	// parse workflow labels
```

Comment:
same here.. this should also be added to remedy workflow.

Reply:
added to remedy workflow.

Reply:
added to remedy workflow